subworkflows/local/analyse_transcripts.nf

```diff
@@ -20,6 +25,18 @@ workflow ANALYSE_TRANSCRIPTS {
         )
         ch_versions = ch_versions.mix(STRINGTIE_STRINGTIE.out.versions.first())
 
+        // DROP
+        // Count file
+        star_count = gene_counts.map{ meta, cnt_file -> cnt_file }.collect()
+        star_samp  = gene_counts.map{ meta, cnt_file -> meta }.collect()
+        GENERATE_COUNTS_DROP(star_count, star_samp, ch_gtf, reference_count_file)
+        ch_versions = ch_versions.mix(GENERATE_COUNTS_DROP.out.versions)
+        // Annotation file
+        GENERATE_ANNOTATION_DROP(GENERATE_COUNTS_DROP.out.processed_gene_counts,ch_gtf,reference_count_file)
+        ch_versions = ch_versions.mix(GENERATE_ANNOTATION_DROP.out.versions)
+
+        //GENERATE_SA4DROP()
```

Comment:
Isn't this what you're doing above :)

Reply:
what do you mean?

Reply:
Meaning that` //GENERATE_SA4DROP()`, isn't that done on line 35 `GENERATE_ANNOTATION_DROP...`